tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "es6",
     "module": "commonjs",
-    "moduleResolution": "node16",
+    "moduleResolution": "nodenext",

```

Comment:
node16 and nodenext both produce this for me now...

```
yield Promise.resolve().then(() => __importStar(require('decompress-response')));
```

But in PROD, I see:
```
yield import('decompress-response');
```

This is only supported on node 13.2+.

Either way, the version is fixed in prod, but TypeScript compilation does worry me. For instance, there isn't a way to offer two options and switch on node version. It's all or nothing.